test/Microsoft.Sbom.Targets.Tests/AbstractGenerateSBomTaskInputTests.cs

```diff
@@ -215,7 +241,11 @@ public void Sbom_Generation_Succeeds_For_Null_Verbosity()
 
         // Assert
         Assert.IsTrue(result);
+#if NET472
+        Assert.IsTrue(this.messages.Any(msg => pattern.IsMatch(msg.Message)));
```

Comment:
Similarly here, we should change all instances of "verbose" to "information" for the default verbosity

Reply:
Fixed at 1e50b1a3bb7c3df73bb2d192f3bdb7ad9ce7e40e